patrol/cli.py

```diff
@@ -20,10 +20,39 @@ def events(organization, project_name):
         click.echo('{eventID}: {message}'.format(**event))
 
 
+@cli.command(help='Lists project events')
+@click.argument('organization')
+@click.argument('project_name')
+@click.argument('event_id')
+def event(organization, project_name, event_id):
+    click.secho('event {} for {}'.format(event_id, project_name))
+    event = patrol.event(organization, project_name, event_id)
+    click.echo('{eventID}: {message}'.format(**event))
+
+
 @cli.command(help='Lists project issues')
 @click.argument('organization')
 @click.argument('project_name')
 def issues(organization, project_name):
     click.secho('issues for {}'.format(project_name))
     for issue in patrol.issues(organization, project_name):
         click.echo('{id}: {title}'.format(**issue))
+
+
+@cli.command(help='Retrive issue')
+@click.argument('issue_id')
+def issue(issue_id):
+    issue = patrol.issue(issue_id)
+    click.echo('{id}: {title} - {status}'.format(**issue))
+
+
+@cli.command(help='Update issue')
+@click.argument('issue_id')
+@click.argument('status')
```

Comment:
I believe here would be better to use a Choice Option http://click.pocoo.org/5/options/#choice-options